### DISCUSSION TITLE
If I use prerender on and ssr off, why when I build I still get a server directory?

### DISCUSSION THREAD

**Discussion by ekiim** [vikejs/vike#1540] [Discussion]
When I run the build process, the `dist` directory contains two directories, `client` and `server`, I've tested the client directory, and it works for static deployment, but I'm wondering about the `server` directory what It is purpose?

Honestly, as a user trying to figure out what vike does and how to use it, it just introduces noise.

Is it totally safe to ignore it and do just a static deployment?

**Comment by brillout** [comment]
I agree and the plan is to remove `dist/server/` but this requires the Vike CLI to be implemented which is work-in-progress.
